docs/sql/array.md

```diff
@@ -42,7 +42,8 @@ as `CREATE TABLE data AS (CITIES VARCHAR ARRAY, COUNTRY VARCHAR)`.
 | `ARRAY_APPEND(` _array_ `,  ` _element_ `)`                          | Appends an element to the end of the array and returns the result. If the array is `NULL`, the function will return `NULL`. If the _element_ is `NULL`, the `NULL` element will be added to the end of the array.             | `ARRAY_APPEND(ARRAY [1, 2], 3)` => `[1, 2, 3]`                      |
 | `ARRAY_JOIN`                                                         | Another name for `ARRAY_TO_STRING`                                                                                                                                                                                            |                                                                     |
 | `ARRAY_LENGTH(` _array_ `)`                                          | Another name for `CARDINALITY`                                                                                                                                                                                                | `ARRAY_LENGTH( ARRAY [1, 2, 3])` => `3`                             |
-| `ARRAY_POSITION(` _array_ `, ` _element_ `)`                         | Returns the (1-based) index of the first _element_ of the _array_ as a long. Returns 0 if _element_ doesn't exist in _array_. If the _element_ is `NULL`, `NULL` is returned.                                                 | `ARRAY_POSITION(ARRAY [1, 3, 4, 6], 4)` => 3                        |
+| `ARRAY_POSITION(` _array_ `, ` _element_ `)`                         | Returns `NULL` if _array_ or _element_ is `NULL`. Returns the (1-based) index of the first _element_ of the _array_ as a long. Returns 0 if _element_ doesn't exist in _array_.                                               | `ARRAY_POSITION(ARRAY [1, 3, 4, 6], 4)` => 3                        |
```

Comment:
there were some recent changes merged in Calcite which will change the behavior of these functions when the array argument is a NULL literal without a cast. But I don't think it's important enough to explain this.